uobtheatre/utils/utils.py

```diff
@@ -34,8 +34,3 @@ def combinations(
         for i in range(1, max_length + 1)
         for combination in itertools.combinations(iterable * i, i)
     )
-
-
-class classproperty(property):  # pylint: disable=invalid-name
```

Comment:
Do we actually want this? Does this fix mypy not knowing what a classmethod property is?